packages/uif/src/indexers/multi/toRanges.test.ts

```diff
@@ -0,0 +1,88 @@
+import { expect } from 'earl'
+
+import { toRanges } from './toRanges'
+import { Configuration } from './types'
+
+describe(toRanges.name, () => {
+  it('empty', () => {
+    const ranges = toRanges([])
+    expect(ranges).toEqual([
+      { from: -Infinity, to: Infinity, configurations: [] },
```

Comment:
I am not a big fan of this -Infinity, in my mind UIF operates <0,Infinity>, maybe there should be an assert during initialization that every configuration is after or equal to "0", and then this function will return nothing less than 0

Reply:
Actually there is nothing in UIF that wouldn't work with negative numbers